conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -29,6 +29,7 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${RECIPE_ROOT}:/recipe_root \
                         -v ${FEEDSTOCK_ROOT}:/feedstock_root \
```

Comment:
I think this was partially inspired by annoying permission issues I had with `docker-machine`. Now that we have narrowed those down, I'm ok with leaving these alone.